TESTS/netsocket/gethostbyname/main.cpp

```diff
@@ -27,20 +27,18 @@
 
 using namespace utest::v1;
 
+namespace {
 // Hostname for testing against
 // Must have A and AAAA records
-#ifndef MBED_DNS_TEST_HOST
-#define MBED_DNS_TEST_HOST "connector.mbed.com"
-#endif
-
-
+const char *DNS_TEST_HOST = "connector.mbed.com";
```

Comment:
Hi
Please check #5576 
I would like to be able to define this host name with MBED app json file.
Thx